tests/test_core.py

```diff
@@ -751,6 +751,66 @@ def test_scaper_init():
     assert sc.fade_out_len == 0.01  # 10 ms
 
 
+def test_scaper_reset():
```

Comment:
Since we're not testing a proper reset, let's rename this `test_reset_fgbg_event_spec()`